app/views/shared/processed_and_deleted/_index.html.slim

```diff
@@ -8,7 +8,7 @@
       table *{class: "#{state}-applications"}
         thead
           tr
-            th =t('processed_applications.table_header.applicant')
+            = table_header
```

Comment:
Shouldn't this be replicated in the data below?  You are changing the heading to say reference, but the link always displays applicant